src/MCPServer/lib/archivematicaMCP.py

```diff
@@ -260,6 +260,74 @@ def cleanupOldDbEntriesOnNewRun():
 
 
+def created_shared_directory_struture():
+    dirs = (
+        "arrange",
+        "completed",
+        "completed/transfers",
+        "currentlyProcessing",
+        "DIPbackups",
+        "failed",
+        "rejected",
+        "sharedMicroServiceTasksConfigs",
+        "sharedMicroServiceTasksConfigs/createXmlEventsAssist",
+        "sharedMicroServiceTasksConfigs/generateAIP",
+        "sharedMicroServiceTasksConfigs/generateAIP/bagit",
+        "sharedMicroServiceTasksConfigs/processingMCPConfigs",
+        "sharedMicroServiceTasksConfigs/transcoder",
+        "sharedMicroServiceTasksConfigs/transcoder/defaultIcons",
+        "SIPbackups",
+        "tmp",
+        "watchedDirectories",
+        "watchedDirectories/activeTransfers",
+        "watchedDirectories/activeTransfers/baggitDirectory",
+        "watchedDirectories/activeTransfers/baggitZippedDirectory",
+        "watchedDirectories/activeTransfers/Dspace",
+        "watchedDirectories/activeTransfers/maildir",
+        "watchedDirectories/activeTransfers/standardTransfer",
+        "watchedDirectories/activeTransfers/TRIM",
+        "watchedDirectories/approveNormalization",
+        "watchedDirectories/approveSubmissionDocumentationIngest",
+        "watchedDirectories/quarantined",
+        "watchedDirectories/SIPCreation",
+        "watchedDirectories/SIPCreation/completedTransfers",
+        "watchedDirectories/SIPCreation/SIPsUnderConstruction",
+        "watchedDirectories/storeAIP",
+        "watchedDirectories/system",
+        "watchedDirectories/system/autoProcessSIP",
+        "watchedDirectories/system/autoRestructureForCompliance",
+        "watchedDirectories/system/createAIC",
+        "watchedDirectories/system/reingestAIP",
+        "watchedDirectories/uploadDIP",
+        "watchedDirectories/uploadedDIPs",
+        "watchedDirectories/watchedDirectories",
+        "watchedDirectories/watchedDirectories/system",
+        "watchedDirectories/watchedDirectories/system/autoProcessSIP",
+        "watchedDirectories/workFlowDecisions",
+        "watchedDirectories/workFlowDecisions/compressionAIPDecisions",
+        "watchedDirectories/workFlowDecisions/createDip",
+        "watchedDirectories/workFlowDecisions/createTree",
+        "watchedDirectories/workFlowDecisions/examineContentsChoice",
+        "watchedDirectories/workFlowDecisions/extractPackagesChoice",
+        "watchedDirectories/workFlowDecisions/metadataReminder",
+        "watchedDirectories/workFlowDecisions/quarantineTransfer",
+        "watchedDirectories/workFlowDecisions/selectFormatIDToolIngest",
+        "watchedDirectories/workFlowDecisions/selectFormatIDToolTransfer",
+        "www",
+        "www/AIPsStore",
+        "www/AIPsStore/transferBacklog",
+        "www/AIPsStore/transferBacklog/arrange",
+        "www/AIPsStore/transferBacklog/originals",
+        "www/DIPsStore"
+    )
+    for d in dirs:
+        d = os.path.join(django_settings.SHARED_DIRECTORY, d)
```

Comment:
Perhaps out of scope for this PR, but how do we know if the django_settings.SHARED_DIRECTORY matches what has been configured in the storage service for this pipeline.  

Maybe the storage service should be invoking these methods, when the currently processing location for a pipeline is set?